### PR TITLE
[RFR]Added ouiaid attributes to all UI components

### PR DESCRIPTION
**Pull request by mayurilahane**

### CODE REVIEW

File: src/SmartComponents/Recs/Details.js [hunk 1]
```diff
@@ -218,17 +219,18 @@ const OverviewDetails = ({ match, fetchRuleAck, fetchTopics, fetchSystem, fetchR
...
+                                        ouiaId="actions"
                                         toggle={<DropdownToggle
                                             isDisabled={!permsDisableRec}
                                             onToggle={(actionsDropdownOpen) => setActionsDropdownOpen(actionsDropdownOpen)}
                                             toggleIndicator={CaretDownIcon}>Actions
                                         </DropdownToggle>}
                                         isOpen={actionsDropdownOpen}
                                         dropdownItems={rule && rule.rule_status === 'enabled' ?
-                                            [<DropdownItem key='link'
+                                            [<DropdownItem key='link' ouiaId="item1"
```
Comment (AllenBW) [inline src/SmartComponents/Recs/Details.js:230]: should these ids be more descriptive? "disable" or "enable"

**Review by AllenBW — DISMISSED** [review]
Looks awesome to me! but will defer to @skarekrow for final approval

File: src/SmartComponents/Recs/Details.js (hunk 1 above)
Comment (skarekrow) [reply src/SmartComponents/Recs/Details.js:230]: I agree, the rest are fine but these are generic and aren't terribly helpful as is.

Comment (mayurilahane) [reply src/SmartComponents/Recs/Details.js:230]: Agree!
I was just not sure what should be the proper id.
Thanks for reviewing :)

**Review by skarekrow — APPROVED** [review]
LGTM on my end @AllenBW thanks @mayurilahane!

**Review by AllenBW — APPROVED** [review]
luv it AWESOME! MERGING 🎉 🌮 🧼

**Pull request merged**